alias-forwarder-email-service/README.md

```diff
@@ -0,0 +1,56 @@
+# A better customer email management solution
+
+This RFC investigates ways for managing (urgent) requests from customers received via email.
+
+## Current setup
+
+In case of urgency, customers can open urgent tickets by sending an email to urgent@giantswarm.io. This will automatically create a ticket in Opsgenie.
+
+In case of non-urgent need, customers can open tickets by sending an email to support@giantswarm.io. This will automatically be forwarded to the #support channel in Slack.
+
+While this works -also thanks to the fact that the tickets are broadcast- we reckon that, as Giant Swarm grows and gathers customers, having such general entry points can become hard to scale and manage.
+
+## Desired improvements
+
+We would like to be able to configure email aliases (where *alias*="some smart way to express and parse a composite email address") with different formats. These aliases should follow an easily parsable and standardized structure, which allows us to redirect the emails to the appropriate target.
+
+For instance, some emails (`urgent`) could be ridirected to Opsgenie, while others (`support`) could be redirected to Slack.
```

Comment:
I'd like a bit more on this. Sure, we could have different emails per customers. Why would we want that? What problem does it solve?

Reply:
I'm still wrapping my head around this and gathering information to try and see whether this would actually be useful.

In my opinion we should be able to solve/improve the following:

- **urgency**: if I understand correctly, an `urgent` email gets forwarded to all the teams/areas simultaneously (https://intranet.giantswarm.io/docs/support-and-ops/processes/oncall). It would be nice to just forward the ticket to the correct team (e.g. if the problem is related to AWS, to the Cloud Area on-caller). Because of this, it would be cool to be able to filter out emails. Again, the idea is to have in somewhere in the email address, but this can change.

- **customer**: we don't really have a use case for this so far, but I can think of some:
  - analytics (therefore understanding what/how we can improve)
  - stopping receiving emails if a customer stops paying
  - maybe prioritizing?

  so we may want to have the information because it *might* be useful in the future. Again, just thinking out loud.

Finally, I will send another update to this PR today with additional questions/information 😄  I'll keep this updated as long as I have news.

Reply:
oh yeah, being able to dispatch emails directly to the right oncall area would be dope, that's legit

Reply:
And the customer makes them feel special and it is an additional trigger that allows us to do things. We could auto populate info for escalation through the wiki. We could do a lot of things.